CLAUDE.md

```
@@ -0,0 +1,35 @@
+# WeatherScope
+
+Android weather app. Single screen. Clean Architecture (domain / data / presentation).
+Package: `com.axehai.weatherscope` | Min SDK 29 | Kotlin + Jetpack Compose + Material3
+
+## Stack
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)